crates/re_ui/src/lib.rs

```diff
@@ -515,6 +515,70 @@ impl ReUi {
         response
     }
 
+    /// Popup similar to [`egui::popup_below_widget`] but suitable for use with
+    /// [`crate::list_item::ListItem`].
+    pub fn list_item_popup<R>(
+        ui: &egui::Ui,
+        popup_id: egui::Id,
+        widget_response: &egui::Response,
+        vertical_offset: f32,
+        add_contents: impl FnOnce(&mut egui::Ui) -> R,
+    ) -> Option<R> {
+        if ui.memory(|mem| mem.is_popup_open(popup_id)) {
```

Comment:
let's be mindful of rightward drift

```suggestion
        if ui.memory(|mem| mem.is_popup_open(popup_id)) {
            return None;
        }
```